engine/src/test/scala/cromwell/WorkflowOutputsSpec.scala

```diff
@@ -63,7 +63,7 @@ class WorkflowOutputsSpec extends CromwellTestkitSpec {
     "Allow explicitly named inputs in the output section" ignore {
```

Comment:
Do we know the provenance of this feature in the first place? Was it actually something that anyone used?